Sources/Config.php

```diff
@@ -757,7 +757,7 @@ class Config
 				 * Path to the Sources directory.
 				 */
 				END,
-			'default' => 'dirname(__FILE__) . \'/Sources\'',
+			'default' => '__DIR__ . \'/Sources\'',
```

Comment:
This is acceptable, but only if the same change is made in ./other/Settings.php.

Reply:
All references to dirname(__FILE__) changed to __DIR__ in Settings.php & Settings_bak.php.